internal/wire/transport_parameters.go

```diff
@@ -481,6 +481,18 @@ func (p *TransportParameters) ValidFor0RTT(saved *TransportParameters) bool {
 		p.ActiveConnectionIDLimit == saved.ActiveConnectionIDLimit
 }
 
+// ValidForUpdate checks that the newly received transport parameters do not contain any redueced limit when resuming a 0-RTT connection
```

Comment:
Should we turn this around? Should `ValidForUpdate` be called on the _new_ transport parameters, and take a `saved` as a paramater? This is how `ValidFor0RTT` is defined.